### ISSUE TITLE
Columns with default values breaks validation and causes queries to return invalid data

### ISSUE THREAD

**Issue by Sinusoidal36** [collerek/ormar#202]
**Describe the bug**
When a model contains a column with a default value this causes queries to run into validation errors. I believe this is because ormar does not fetch the whole state of related models but rather only returns the primary key unless requested explicitly with .select_related(). A model with defaults breaks this behavior and instead returns the primary key and populates default values, excluding other required fields.

**To Reproduce**
```python
from typing import Optional
from fastapi import FastAPI

import databases
import pydantic

import ormar
import sqlalchemy

DATABASE_URL = "sqlite:///bug.sqlite"
database = databases.Database(DATABASE_URL)
metadata = sqlalchemy.MetaData()

class BaseMeta(ormar.ModelMeta):
    metadata = metadata
    database = database

class Author(ormar.Model):
    class Meta(BaseMeta):
        tablename = "authors"

    id: int = ormar.Integer(primary_key=True)
    name: str = ormar.String(max_length=100)
    rating: int = ormar.Integer(default=0) #<------------- default value

class Book(ormar.Model):
    class Meta(BaseMeta):
        tablename = "books"

    id: int = ormar.Integer(primary_key=True)
    author: Optional[Author] = ormar.ForeignKey(Author)
    title: str = ormar.String(max_length=100)
    year: int = ormar.Integer(nullable=True)

engine = sqlalchemy.create_engine(DATABASE_URL)
# just to be sure we clear the db before
metadata.drop_all(engine)
metadata.create_all(engine)

author = Author(name='bug', rating=5)
book = Book(author=author, title='Bug caused by default value', year=2021)

app = FastAPI()

@app.on_event('startup')
async def startup():
    await author.save()
    await book.save()

@app.get('/{id}', response_model=Book)
async def get_book_by_id(id: int):
    book = await Book.objects.get(id=id)
    print(book)
    return book
```
Running the above code, then requesting the book we created causes a validation error:
```python
pydantic.error_wrappers.ValidationError: 1 validation error for Book
response -> name
  field required (type=value_error.missing)
```
The print out of the book object before returning the response is:
```
id=1 author=Author({'id': 1, 'name': None, 'rating': 0, 'books': [<weakproxy at 0x7f476effaa40 to Book at 0x7f47700a3b20>]}) title='Bug caused by default value' year=2021
```

As you can see here, the instance of Author in the book we retrieved has invalid state. Its 'name' value is None and has the default rating value populated, but that violates the pydantic models validation which occurs when FastAPI constructs the response.

**Expected behavior**
I expect ormar to not return these default values for related models. This is extra dangerous because you could mistakenly think that the default values returned reflect the actual state of the object.

**Versions**
 - Database backend used **sqlite**
 - Python version **3.9.4**
 - `ormar` version **0.10.5**
 - `pydantic` version **1.8**
 - `fastapi` version **0.63.0**

**Comment by collerek** [comment]
Yeah that should be a pk only model with just pk populated, something is wrong here, will check that.

Thanks for catching that 🙂

**Comment by collerek** [comment]
Fixed in 0.10.8 (#203)

**Comment by Sinusoidal36** [comment]
Wow, quick fix! Thanks for the useful python package.
